tests/fake_texture.rs

```diff
@@ -0,0 +1,18 @@
+extern crate graphics;
+
+use self::graphics::ImageSize;
+
+pub struct FakeTexture {
```

Comment:
[here is the docs for 'unit structs'](http://doc.rust-lang.org/stable/book/structs.html#unit-like-structs)